snapshot/lib/snapshot/options.rb

```diff
@@ -181,6 +181,11 @@ def self.available_options
                                      env_name: "SNAPSHOT_NAMESPACE_LOG_FILES",
                                      description: "Separate the log files per device and per language",
                                      optional: true,
+                                     is_string: false),
+        FastlaneCore::ConfigItem.new(key: :concurrent_simulators,
+                                     env_name: "SNAPSHOT_EXECUTE_CONCURRENT_SIMULATORS",
+                                     description: "Take snapshots on multiple simulators concurrently",
```

Comment:
Maybe add a note to the `description` that this only works on Xcode 9 and above?